cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/ShuntCompensatorToShuntCompensator.java

```diff
@@ -15,7 +15,6 @@
 public class ShuntCompensatorToShuntCompensator extends IidmToCgmes {
 
     ShuntCompensatorToShuntCompensator() {
-        ignore("q");
 
         simpleUpdate("bPerSection", "cim:LinearShuntCompensator.bPerSection", CgmesSubset.EQUIPMENT);
```

Comment:
Now, we have non linear shunt compensator, so it means that we also have to support that for the CGMES update. I wonder if the code is working for that kind of equipment.

Reply:
I will have to review. Current export does not support `NonLinearShuntCompensators`.

Reply:
I propose to add export support for that equipment in a separate, specific PR.

Reply:
Yes, great. I am going to open an issue to keep that in mind and I am going to merge this PR as it is.